src/components/elements/Catalog.js

```diff
@@ -1,34 +1,47 @@
-import React from 'react';
-import { GridList } from '@material-ui/core';
-import GridListTile from '@material-ui/core/GridListTile';
-import { Search } from './Search';
-import "./css/Catalog.css"
-import { ParkCard } from './ParkCard';
-import Header from './Header';
-
+import React from "react";
+import { GridList } from "@material-ui/core";
+import GridListTile from "@material-ui/core/GridListTile";
+import { Search } from "./Search";
+import "./css/Catalog.css";
+import { ParkCard } from "./ParkCard";
+import axios from "axios";
 
 export class Catalog extends React.Component {
```

Comment:
Any reason for keeping class react components when you are using functional components in other parts of your implementation?